src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,35 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private final List<EventListener> listeners = new CopyOnWriteArrayList<>();
+    private final ExecutorService executor = Executors.newCachedThreadPool();
```

Comment:
Using 'newCachedThreadPool' can be risky as it can create a large number of threads if many events are received in a short time. This can lead to performance issues. If you expect a high volume of events, consider using a fixed-size thread pool or another type of executor that limits the number of threads.